Code_of_Conduct.md

```diff
@@ -0,0 +1,33 @@
+# Contributor Code of Conduct
+
+## Introduction
+
+As contributors, we work to foster a welcoming and inclusive community. By making this project open to public, we embrace conversation around the topic and encourage people of diversities to fork it and incorporate new insights. Meanwhile, we pledge to respect everyone participating in this project, regardless of race, ethnicity, nationality, body size, religion, disability, gender identity and expression, political affiliation, socioeconomic status, or experience in the topic.
+
+## Expected Behavior
+
+- Communicating using welcoming and inclusive languages
+- Respecting and embracing diversified opinions
+- Giving constructive and generous critiques
+- Supporting collaborators by giving assistance
+
+## Unacceptable Behavior
+
+- Using sexualized language or imagery
+- Harassing any participant in any form
+- Insulting comments
+- Personal, religious, or political attack
+- Publishing other individuals' private information, such as phone numbers or email addresses, without explicit permission
+- Other inappropriate, unwelcome or unprofessional conduct
+
+## Consequences
+
+If any participant violates this code of conduct and engages in any unacceptable behavior, project maintainers have the right and responsibility to take fair and proper action against that, including editing, removing or deleting the comments, issues or other forms of contribution temporarily or permanently, and banning the participants from contributing for behaviors deemed as offensive and inappropriate. 
+
+## Reporting & Enforcement
+
+If you believe someone is violating this code of conduct or have other concerns, please report by contacting one of the project maintainers at wanying.ye2020@gmail.com. The project maintainer will review and respond in a way that deems appropriate to the circumstances. Messages sent to this email address will not be revealed to anyone except for the project maintainer team to protect the confidentiality of the report of an incident. 
+
```

Comment:
Thanks Gloria. You did an excellent job. Regarding Reporting & Enforcement, I guess we can come up with an idea of how we can solve the conflicts among ourselves in case they happen. For instance, we can say that if a conflict happens, we can take votes. Or, we can mention that if there is a conflict that we cannot solve, we will ask the lab instructor or TAs to help us.

Reply:
Thanks, Paniz. Do you mean adding the actions to take when we have different opinions on some aspects upon doing this project?

I think this document means to clarify what people (e.g. the public and not just us) can do and cannot do with this repo; on the other hand, if we four have conflicts on some aspect in this project, we can discuss and open a pull request etc., which is among the rules we specify in the team contract. 

However, the above is just my opinion after reading the guidelines to create a Code of Conduct and some samples of it. This is open to discussion, and I will incorporate the points if we all agree upon.